logic.py

```diff
@@ -116,11 +122,20 @@ def get_paste_listing(config):
         return "ERROR", "Paste listing has been disabled by the " +\
             "administrator.", 503
 
+    b = config['b']
```

Comment:
I've changed this one to stay below the 80 characters per line limit. I don't think we really need to convert all occurrences as a matter of policy.